package.json

```diff
@@ -34,6 +41,7 @@
     "github-url-from-git": "^1.5.0",
     "github-url-from-username-repo": "^1.0.2",
     "giturl": "^1.0.0",
+    "jest": "^21.3.0-beta.14",
```

Comment:
The beta includes a newer version of jsdom which comes with data attr support 

https://github.com/tmpvar/jsdom/issues/961
https://github.com/facebook/jest/pull/5016#issuecomment-351550634